<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>326</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QScrollArea" name="scrollArea_2">
        <property name="styleSheet">
         <string notr="true">background-color:#f3f6fb;border-radius:10px;
</string>
        </property>
        <property name="widgetResizable">
         <bool>true</bool>
        </property>
        <widget class="QWidget" name="scrollAreaWidgetContents_3">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>306</width>
           <height>580</height>
          </rect>
         </property>
         <widget class="QScrollArea" name="scrollArea">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>510</y>
            <width>305</width>
            <height>71</height>
           </rect>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="styleSheet">
           <string notr="true">border-color: rgb(255, 255, 255);</string>
          </property>
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="scrollAreaWidgetContents_2">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>305</width>
             <height>71</height>
            </rect>
           </property>
           <widget class="QPushButton" name="pushButton">
            <property name="geometry">
             <rect>
              <x>240</x>
              <y>0</y>
              <width>61</width>
              <height>30</height>
             </rect>
            </property>
            <property name="layoutDirection">
             <enum>Qt::LeftToRight</enum>
            </property>
            <property name="autoFillBackground">
             <bool>false</bool>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: rgb(51, 71, 255);</string>
            </property>
            <property name="text">
             <string>Enviar</string>
            </property>
            <property name="icon">
             <iconset resource="sources.qrc">
              <normaloff>:/sources/arrow-34-24.ico</normaloff>:/sources/arrow-34-24.ico</iconset>
            </property>
           </widget>
           <widget class="QLineEdit" name="lineEdit">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>0</y>
              <width>241</width>
              <height>30</height>
             </rect>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: rgb(246, 255, 219);</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="maxLength">
             <number>32767</number>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
            <property name="placeholderText">
             <string>  Enter your message here</string>
            </property>
            <property name="cursorMoveStyle">
             <enum>Qt::LogicalMoveStyle</enum>
            </property>
           </widget>
           <widget class="QPushButton" name="pushButton_2">
            <property name="geometry">
             <rect>
              <x>120</x>
              <y>40</y>
              <width>75</width>
              <height>23</height>
             </rect>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: rgb(97, 131, 255);</string>
            </property>
            <property name="text">
             <string>Fale</string>
            </property>
            <property name="icon">
             <iconset resource="sources.qrc">
              <normaloff>:/sources/microphone-5-24.ico</normaloff>:/sources/microphone-5-24.ico</iconset>
            </property>
           </widget>
          </widget>
         </widget>
         <widget class="QLabel" name="label">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>0</y>
            <width>291</width>
            <height>471</height>
           </rect>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="sources.qrc"/>
 </resources>
 <connections/>
</ui>
